subprojects/kotlin-dsl-provider-plugins/src/main/kotlin/org/gradle/kotlin/dsl/provider/plugins/precompiled/tasks/GeneratePrecompiledScriptPluginAccessors.kt

```diff
@@ -75,6 +76,10 @@ import java.nio.file.Files
 import javax.inject.Inject
 
 
+internal
+val strictModeSystemPropertyName = "org.gradle.kotlin.dsl.precompiled.accessors.strict"
```

Comment:
Should this string be moved somewhere the tests can access it, too?

Reply:
I did not find any reasonable home and thought it's not that bad as it is.
Do you feel strongly about this?

Reply:
Not strongly, and it's just test code.  I didn't know if there was a more appropriate place that already existed.